README.md

```diff
@@ -57,6 +57,8 @@ Also provides conveniences for creating value objects, method objects, query met
 
 `attr_initialize [:bar, :baz!]` defines an initializer that takes one hash argument, assigning `@bar` (optional) and `@baz` (required).
 
+`attr_initialize [ :foo, bar: "baz" ]` defines an initializer that takes one hash argument, assigning `@foo` (optional) and `@bar` (optional with a default value of `"baz"`).
```

Comment:
I wonder if it's worth mentioning that the default value will be evaluated whether or not it is used. Since regular defaults don't work that way:

```
>> def foo(bar: p("hi")); end
=> :foo
>> foo
"hi"
=> nil
>> foo(bar: "what")
=> nil
```